### PR TITLE
Feat restaurant

### PR DESCRIPTION
**Pull request by omar-labana**
Highlights on this pull request;
- Add webpack and Tailwind CSS.
- Add shared module to create elements.
- Add module for each page

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@omar-labana  Great job done by you, Almost all the project requirement is met, However, few changes are required to get your project approved:
### highlights
Git Flow 👍🏻 
entry point named index.js 👍🏻 
it uses Webpack 👍🏻 
 ### Improve
- [ ] Your Installation under the `Getting Started` section is enough descriptive enough to run your app, kindly start how to run your app and also tell us what to do after ` git clone `. fix it.

- [ ] Necessary run common commands like `build`, `development` should be in `package.json` inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)). kindly add it

```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

### once specific changes are made kindly request for review

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "I hate webpack",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (taiwo2) [inline package.json]: Should have Necessary run common commands like build, development should be in package.json inside `scripts'

**Review by taiwo2 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "I hate webpack",
+  "private": true,
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
```
Comment (oloomoses) [inline package.json]: - [ ] Kindly ensure that `webpack-dev-server` is installed under the **devDependencies** in `package.json` for this line to execute.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
- [ ] For this line `webpack-dev-server --mode development --open` Kindly change the `webpack-dev-server` to `webpack serve`. It should read `webpack serve --mode development --open`. This is one of the recent changes that was made to webpack.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @omar-labana, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [ ] It would be better to deploy the app and add the live link in the readme
- [ ] It would be better to include the author's details in the readme as it will make your readme more descriptive

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Slack:  _Oloo Moses_

------

**Review by oloomoses — APPROVED** [review]
Hi @omar-labana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_

**Pull request merged**